cmd/grpc-server/pb/user.proto

```diff
@@ -4,21 +4,23 @@ option go_package = "pb";
 
 
 message UserRequest{
-    string id =1;
-    string name =2;
-    string pwd_hash = 3;
-    int32 age = 4;
-    string additional_information = 5;
-    repeated UserResponse parents =6;
+    int32 user_id =1;
+    string pwd_hash = 2;
+    string email = 3;
+    string name =4;
+    int32 age = 5;
+    string additional_information = 6;
+    repeated UserRequest parents =7;
```

Comment:
Consider using skipped indexes (1, 3, 5, ...). This may help if you need to add or modify a field.